app/src/main/java/com/example/lastfm/MainActivity.java

```diff
@@ -4,81 +4,128 @@
+//                   new NewsData("Новости мира музыки","https://i.imgur.com/QsMxb2f.jpg","Как показывает статистика мало кого вообще  волнуют новости из мира музыки."),
+//                   new NewsData("Летова нашли прячущимся в Тайге","https://ndn.info/images/dimensions/xl/ac8717f117172c129a27516cb01b2828_XL.jpg","Непонятно откуда у него кот, но надеюсь он его кормит"),
+//                   new NewsData("Ниша новостей из сферы музыки скудна и однообразна, заявляют эксперты","https://bipbap.ru/wp-content/uploads/2019/10/Grustnye-15.jpg","И правда, очень мало интересных новостей"),
+//                   new NewsData("Шокирующие новости! Чтобы увидеть настоящие новости...","https://pbs.twimg.com/profile_images/1189275922240811009/kjVm1yVY_400x400.jpg","Нужно всего лишь пролистать вниз"),
+//                   new NewsData("Люди которые перешли с Windows на Linux","https://sun9-35.userapi.com/c837637/v837637260/5942d/QKzye39XIgI.jpg","Все как один отмечают, что их кожа стала мягче, лицо моложе, а играть музыку они стали виртуозно!"),
+//                   new NewsData("Горшок ЖИВ!","https://funeralportal.ru/upload/iblock/bfc/403693.jpg","Да шутка повторилась но хуже от этого не стала."),
+//                   new NewsData("Last.fm","https://www.iguides.ru/upload/medialibrary/c3b/c3b22e87226ce45d6100a9ab0f9942dc.png","Last.fm живет и процветает, осталось только добавить интеграцию в ЯндексМузыку тогда заживем!"),
+//                   new NewsData("ЯндексМузыка за 99 рублей!","https://yt3.ggpht.com/a/AGF-l7_3L_I5maGpVKDugzwEPEak9al3ufizsrkUZw=s900-c-k-c0xffffffff-no-rj-mo","Акция невиданной щедрости, аж в два раза дешевле!"),
+//                   new NewsData("K-POP Переоценен!","https://cdn.eksmo.ru/v2/ITD000000001043430/COVER/cover1__w600.jpg","Не слушал, но смею судить."),
+//                   new NewsData("Что стало с Т.А.Т.У.?","https://n1s1.hsmedia.ru/d4/f0/36/d4f03680616433d136e26a755a1aaa88/2143x1295_0x0a330ca2_13653060181557641052.jpeg","Никто не знает, но каждый переодически переслушивает их песни."),
+//                   new NewsData("Рецензия на клип: Вася Обломов - «Диги-диги»","https://cdn1.intermedia.ru/img/news/344756.jpg","Песня «Диги-диги» входила в недавний альбом Васи Обломова «Этот прекрасный мир» и была там одной из самых милых и мелодичных. Хотя «Диги-диги» и является сокращением от «дегенератов», в устах Васи это звучит почти нежно - что-то типа интонаций, с которой произносится фраза «они же дети». Клип на эту песню Обломов режиссировал сам и не стал экранизировать композицию близко к её тексту («они по левой полосе при свободных правых, у них парковка на газонах для самых-самых, они и делают ремонт, и дома строят, их услуги говно, они дорого стоят»). Вместо этого Вася Обломов, сосредоточился, на, так сказать, определённом виде дегенератов - тех, кто посещает тренинги личностного роста.")
```

Comment:
remove commented code if it's unnecessary

Reply:
Deleted unnecessary part